react/features/chat/components/native/styles.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import {
+    createStyleSheet
+} from '../../../base/styles';
+
+/**
+ * The styles of the feature chat.
+ */
+export default createStyleSheet({
+    modalBackdrop: {
```

Comment:
Why do we use a different value than other modals here?

Reply:
Actually it's the same value at the moment (see https://github.com/jitsi/jitsi-meet/blob/71563fec5b10b6e4ce0ecf27fc12a98cc0dd65c2/react/features/base/dialog/components/native/styles.js#L178), but I don't want to tie it to them, because this is not really a backdrop in the same meaning, but it's the entire chat window, whose transparency we may (likely) want to change to fit to our (or others) branding.

Reply:
I disagree. Consistency is a good thing. Why don't we see how to cross that river if/when we get there?